### DISCUSSION TITLE
Evaluate Browserstacks Github Action

### DISCUSSION THREAD

**Discussion by daKmoR** [modernweb-dev/web#713] [Ideas]
there is an official browstack github action now.

maybe we should make use of it? 🤔

https://www.browserstack.com/docs/automate/selenium/github-actions

I am however not sure how well it fits in the current workflow/setup of web test runner 🤔

@LarsDenBakker what do you think?

**Comment by LarsDenBakker** [comment]
It looks like it sets up the browserstack proxy, which we already do inside the test runner. We could make that behavior configurable, but not sure if there is any added value.

**Comment by daKmoR** [comment]
jup that's sort of my question - would this github action give better feedback in the github UI or is there any benefit? because if not then we won't need it
